api/src/lib/ai/prompts.ts

```diff
@@ -302,6 +308,176 @@ Use the tool "make_request". Always respond in valid JSON.
 ***Don't make your responses too long, otherwise we cannot parse your JSON response.***
 `);
 
+export const LLAMA_3_8B_FRIENDLY_PARAMETER_GENERATION_SYSTEM_PROMPT =
```

Comment:
this prompt has been modified from the OG system prompt we usually send to anthropic and openai

i found that reducing the scope of what we're trying to do worked better. so, we ONLY tell it to deal with json bodies, not worry about text + form data + etc